app/pandas.ipynb

```diff
@@ -271,54 +601,197 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "outputs": [],
    "source": [
     "# Data\n",
     "data1 = {'Name': ['John', 'Emily', 'Ryan'],\n",
     "         'Age': [25, 28, 22]}\n",
     "data2 = {'Name': ['Emily', 'Ryan', 'Mike'],\n",
     "         'City': ['Los Angeles', 'Chicago', 'Houston']}\n",
     "\n",
-    "# create DataFrame objects"
+    "dat1 = pd.DataFrame(\n",
+    "    data1\n",
+    ")\n",
+    "dat2 = pd.DataFrame(\n",
+    "    data2\n",
+    ")\n"
    ],
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-05T09:41:32.371132Z",
-     "start_time": "2023-07-05T09:41:32.366179Z"
+     "end_time": "2024-11-08T13:21:51.744575Z",
+     "start_time": "2024-11-08T13:21:51.739146Z"
     }
-   }
+   },
+   "outputs": [],
+   "execution_count": 49
   },
   {
    "cell_type": "code",
-   "execution_count": 22,
-   "outputs": [],
    "source": [
-    "# merge"
+    "merg = pd.merge(dat1, dat2)\n",
+    "merg"
    ],
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-05T09:41:41.152019Z",
-     "start_time": "2023-07-05T09:41:41.137327Z"
+     "end_time": "2024-11-08T12:54:04.435082Z",
+     "start_time": "2024-11-08T12:54:04.407172Z"
     }
-   }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "    Name  Age         City\n",
+       "0  Emily   28  Los Angeles\n",
+       "1   Ryan   22      Chicago"
+      ],
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>Name</th>\n",
+       "      <th>Age</th>\n",
+       "      <th>City</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>Emily</td>\n",
+       "      <td>28</td>\n",
+       "      <td>Los Angeles</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>Ryan</td>\n",
+       "      <td>22</td>\n",
+       "      <td>Chicago</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ]
+     },
+     "execution_count": 22,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 22
   },
   {
    "cell_type": "code",
-   "execution_count": 24,
-   "outputs": [],
    "source": [
-    "# concatenate"
+    "con = pd.concat([dat1, dat2])\n",
+    "con"
```

Comment:
Consider using `ignore_index=True` in the `pd.concat()` function to reset the index of the concatenated DataFrame.